tests/Application/templates/bundles/SyliusShopBundle/_scripts.html.twig

```diff
@@ -5,7 +5,6 @@
 
 <script type="text/javascript"
         src="https://static-sogecommerce.societegenerale.eu/static/js/krypton-client/V4.0/stable/kr-payment-form.min.js"
-        integrity="sha384-FIqRg5Z3CArcz1UvV7osFqEHYW01awqNCTwwTYoL6lhBgo7+3t9yswE3vO93wOGv"
```

Comment:
Question: Why delete integrity attribute ? Especially for a payment solution ?

Reply:
The integrity changed since the PR was opened.. I don't think they have a fixed version URL.

https://sogecommerce.societegenerale.eu/doc/en-EN/rest/V4.0/javascript/spa/

The documentation itself don't use the integrity